one_fm/overrides/leave_application.py

```diff
@@ -158,15 +158,16 @@ def custom_notify_employee(self):
         parent_doc = frappe.get_doc("Leave Application", self.name)
         args = parent_doc.as_dict()
         email_template = frappe.get_doc("Email Template", template)
-        message = frappe.render_template(email_template.response, args)
+        message = frappe.render_template(email_template.response_html, args)
```

Comment:
Please attach a screenshot to the form

Reply:
I don't understand this

Reply:
![image](https://github.com/user-attachments/assets/582fc0b4-a8ae-42ea-b1bf-0e2f80c37e20)